dev-tools/mage/fields.go

```diff
@@ -143,13 +143,33 @@ func GenerateModuleFieldsGo(moduleDir string) error {
 // GenerateModuleIncludeListGo generates an include/list.go file containing
 // a import statement for each module and dataset.
 func GenerateModuleIncludeListGo() error {
-	return GenerateIncludeListGo(nil, []string{"module"})
+	return GenerateIncludeListGo(nil, []string{"module"}, nil, "include/list.go", "")
+}
+
+// GenerateOSSMetricbeatModuleIncludeListGo generates include/list_{suffix}.go files containing
+// a import statement for each module and dataset.
+func GenerateOSSMetricbeatModuleIncludeListGo() error {
```

Comment:
I think this metrcibeat-specific function should go into `metricbeat/scripts/mage`.